.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: bash
+services:
+  - docker
+addons:
+  apt:
+    packages:
+      - docker-ce
+sudo: required
+
+env:
+  global:
+    - DOCKER_IMAGE=mcguinness/saml-idp
+
+before_install:
+  - tagCommit=`echo ${TRAVIS_COMMIT} | sed "s|/|-|"`
+  - export DOCKER_IMAGE_COMMIT=${DOCKER_IMAGE}:${TAG_COMMIT}
+
+script: docker build . -t ${DOCKER_IMAGE_COMMIT}
+
+before_deploy:
+  - export DOCKER_IMAGE_TAG=${DOCKER_IMAGE}:${TRAVIS_TAG}
+  - echo "${DOCKER_PASSWORD}" | docker login -u "${DOCKER_USERNAME}" --password-stdin
```

Comment:
`DOCKER_USERNAME` and `DOCKER_PASSWORD` need to be set by a repo owner after setting up on travis and docker hub.

See https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings

Reply:
@jacobcabantomski-ct mind hard forking this and setting up the publicly-available Docker image on your fork? I'd be interested in this too and clearly the author is not maintaining this repo anymore.